packages/express-openapi/test/sample-projects/with-customKeywords/spec.js

```
@@ -0,0 +1,40 @@
+var app;
+var request = require('supertest');
+
+before(function () {
+  app = require('./app.js');
+});
+
+describe('Testing type coercion using custom keywords', function () {
+  it('Custom keyword "x-coerce" should convert millis since unix-epoch to Date object', function (done) {
+    let now = new Date();
+    request(app)
+      .get(`/v3/coerce?milliseconds=${now.getTime()}&ignore=foo`)
+      .expect(
+        200,
+        { isoDateString: now.toISOString(), ignoredString: 'foo' },
+        done
+      );
+  });
+
+  it('Custom keyword "x-coerce" should fail on invalid input data', function (done) {
```

`it('should fail when the data is invalid'...`